src/forder.c

```diff
@@ -1275,9 +1275,10 @@ SEXP isReallyReal(SEXP x) {
   // which needed a repeat of the argument. Hence simpler and more robust to return 0 when not type double.
   if (isReal(x)) {
     int n=length(x), i=0;
+    double *dx = REAL(x);
```

Comment:
This is likely to speed up code in few other places, sometimes speed up might be significant.